vng_api_common/middleware.py

```diff
@@ -76,7 +76,10 @@ def payload(self) -> dict:
                 code='invalid-jwt-signature'
             )
 
-        return payload.get('zds', EMPTY_PAYLOAD)
+        res = payload.get('zds', EMPTY_PAYLOAD)
+        res['client_id'] = payload.get('iss', '')
```

Comment:
this should use the client id from the header:
```suggestion
        res['client_id'] = header['client_identifier']
```

Reply:
fixed